### DISCUSSION TITLE
[Feature] uploaded client identity uses in store template

### DISCUSSION THREAD

**Discussion by mochi-c** [immich-app/immich#14921] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

is it possible to record the uploaded client identity when backup (upload) the file? and mark it as a variable available in the store template?

### Platform

- [x] Server
- [x] Web
- [x] Mobile

**Comment by bo0tzz** [comment]
#5687
